index.src.html

```diff
@@ -374,22 +397,30 @@ <h3 dfn export id="referrer-policy-strict-origin">"<code>strict-origin</code>"</
     <code>http://example.com/</code>.
   </div>
 
+  <div class="example">
+    If a document at <code>http://example.com/page.html</code> sets a policy of
+    <a>"<code>strict-origin</code>"</a>, and fetches a module script at
+    <code><strong>https</strong>://script.example.com</code>, which then fetches a
+    descendant script at <code><strong>http</strong>://descendant.example.com</code>,
```

Comment:
I think that almost all browsers would prevent this fetch from happening as a mixed script load (unless overridden by the user). It thus might be a slightly confusing example to include and perhaps not super useful (since it wouldn't come up often in real life). I'm open to being convinced otherwise though.

Reply:
That's what I thought initially, when I wrote tests for this a while ago, you can see in Chrome's console that if an secure module imports an insecure descendant, the descendant is not blocked. Visit http://wpt.live/html/semantics/scripting-1/the-script-element/module/referrer-strict-policies.sub.html and see the console in at least Chrome and Safari. The request doesn't seem to be blocked.

Reply:
Ohh, I missed that the top-level document in this example is http://. I'd expect it to be blocked when the top-level document is https://, but not http://, so this makes sense.